pkg/controllers/report/background/controller.go

```diff
@@ -278,7 +299,7 @@ func (c *controller) reconcileReport(
 	// calculate necessary results
 	for _, policy := range backgroundPolicies {
 		if full || actual[reportutils.PolicyLabel(policy)] != policy.GetResourceVersion() {
-			scanner := utils.NewScanner(logger, c.engine, c.config, c.jp)
+			scanner := utils.NewScanner(logger, c.engine, c.config, c.jp, c.policies, c.vapPolcies)
```

Comment:
You don't need `c.vapPolicies` here since we're looping through kyverno policies.